src/components/progress/progress.test.tsx

```diff
@@ -15,4 +15,18 @@ describe('<Progress /> component', () => {
         render(createTestComponent());
         expect(screen.getByRole('progressbar')).toBeInTheDocument();
     });
+
+    describe('variant tests', () => {
+        it('renders medium variant correctly', () => {
+            render(createTestComponent({ variant: 'md' }));
+            const progressBar = screen.getByRole('progressbar');
+            expect(progressBar).toHaveClass('h-[8px]');
+        });
+
+        it('renders small variant correctly', () => {
+            render(createTestComponent({ variant: 'sm' }));
+            const progressBar = screen.getByRole('progressbar');
+            expect(progressBar).toHaveClass('h-[4px]');
+        });
+    });
```

Comment:
We don't usually need to check the classes of a component through `toHaveClass`, we shouldn't need to update unit tests when changing the style of a component. We try to write tests for all component functionalities but not for its styles 
```suggestion
```